guillotina/api/service.py

```diff
@@ -36,6 +42,56 @@ def __setattr__(self, name, value):
 
 
 class Service(View):
+    __validator__ = None
+    _sentinal = object()
+
+    def _get_validator(self, config):
```

Comment:
should we add type annotations?

Reply:
I am sorry, What is annotation?

Reply:
yep

Reply:
`def _get_validator(self, config = None):`